hazelcast/src/main/java/com/hazelcast/map/impl/MapStoreWrapper.java

```diff
@@ -99,7 +100,8 @@ public void deleteAll(Collection keys) {
 
     public Iterable loadAllKeys() {
         if (isMapLoader()) {
-            return mapLoader.loadAllKeys();
+            // Invoke reflectively to preserve backwards binary compatibility
+            return ReflectionHelper.invokeMethod(mapLoader, "loadAllKeys");
```

Comment:
indeed, I like that. means it'll be easier to navigate the call hierarchy in IDE as well